flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ArchivedExecutionGraphTest.java

```diff
@@ -159,19 +149,11 @@ public void testSerialization() throws IOException, ClassNotFoundException {
         verifySerializability(archivedGraph);
     }
 
-    @Test
-    public void testCreateFromInitializingJobForSuspendedJob() {
```

Comment:
Moving around these methods makes hard to compare the diff of the changes.

Reply:
It must be a mistake. Sorry for that and I'll move them back.